### ISSUE TITLE
Clarification about how to don the blindfold

### ISSUE THREAD

**Issue by EdHollingdale** [thewca/wca-regulations#702]
B4a states that "The competitor dons the blindfold to start the blindfolded phase" however we are not specific about how this has occurs. In practice people have in the past used one hand to push the blindfold over their eyes before starting execution. 

In the last few days there has been discussion about the "nod don" where a competitor can don the blindfold by nodding their head, thus saving a fraction of a second. Given that B4a does not offer guidance about how the blindfold should be donned, this seems like it would be allowed. Should this be the case? Either way I feel an additional clarification in the guidelines might be useful here?

**Comment by pedrosino** [comment]
Where did this discussion happen? Can you link or copy the most relevant stuff here?

I don't think we need to regulate this kind of stuff. As long as the blindfold is correctly placed, how people do it shouldn't matter.

**Comment by EdHollingdale** [comment]
It was posted in a private facebook group that many cubers nonetheless would have access to: https://www.facebook.com/groups/cyoubxfriends/permalink/2648228388536931/ 

It was mainly a video showing the technique. Some discussed that this makes it difficult for the judge to clearly tell when the attempt moves from inspection to execution. However the nod itself is obvious enough I feel.

**Comment by xsrvmy** [comment]
Here is a public video: https://youtu.be/UyCMU83kXkQ

**Comment by mark49152** [comment]
Per B4e, the competitor can return to the memorization phase at any time before the first move is made.  It might make more sense to define the start of the execution phase to be the first move, instead.  That is the point of no return, at which point the judge should ensure the blindfold is down and the sight blocker is present.  IMHO it doesn't really matter how the blindfold is donned as long as it is present for the first move and beyond.

**Comment by lgarron** [comment]
> IMHO it doesn't really matter how the blindfold is donned as long as it is present for the first move and beyond.

I agree with this; the problem isn't how they don the blindfold but whether they've donned it.

It might actually be nice to see something like cozy goggles with a front that folds down. That can be convenient for the competitor *and* is clearly blocking sight.

**Comment by ChrisBLD** [comment]
Hey all,

I'd like to formally give my opinions on this and discuss some things I've noticed having successfully used nod-don to get my 20.67 single UK NR last weekend. On my [YouTube channel](https://www.youtube.com/channel/UCX4LzNj0n6Pq1VfAjglln0A) I have two videos from the competition, one being my NR single and the other being a 24.65 from the previous round which gives a much better angle of the usage of nod-don than my NR does.

First, I spoke with Jules at the competition after he had seen the video of my record, where I had a decent discussion with him regarding my thoughts on this, which I'll now put on record.

TL;DR - Despite what appears to be popular belief, I think this should be disallowed in the future. I'm really on the fence but I think if it came down to it I'd have to err on the side of banning this technique. That being said, I definitely appreciate the reasons given by both sides on this argument which makes it really difficult to come to this conclusion.

I'll give some of my biggest concerns with using this technique first. I'll be referring to nod-don as ND for the remainder of this post.

**1) The ambiguity that already technically exists between the end of the memorisation phase and the beginning of the solving phase is drastically increased using ND.** 

The argument I see brought up a lot when somebody mentions the ambiguity that ND causes is that "regular 3BLD has this ambiguity already since somebody could make moves with one hand whilst donning the blindfold with the other". Whilst I accept this argument I think that comparing the two situations side by side is a little disingenuous. On the one hand, you are presented with the ability to start your execution with your hands already holding the cube exactly as you need to perform your first commutator at full speed as soon as the blindfold is donned. On the other, you get to perform maybe two or three moves maximum (and considering most top BLD solvers solve edges first using commutators that involve E/S/M moves this really isn't feasible at all as a time saving strategy) whilst in the process of returning your other hand to the puzzle. I feel like if there was a way to leverage this to your advantage then people would have done so by now (although you never really know considering this is the first time ND has been properly practiced by World Class BLD solvers such as Jack and Graham).

What are the consequences of this elevated ambiguity? I think they are far too concerning to ignore.

Some preface to this: The people who are the top of their game are implementing ND because saving even 0.3 seconds at sub-25 level is an insane amount of time, especially for something so minor. We're talking about people who learn entire new buffers to save fractions of seconds of time here. People look to these solvers as role models and will begin to implement ND themselves even if they average upwards of 2 minutes. In the same way that people who average above 5 minutes on 7x7 go and buy the Spark when they're currently using the Hays7, the optimisations made by the elite solvers trickle down into the whole community.

**Comment by ChrisBLD** [comment]
Hey all,
I'd like to formally give my opinions on this and discuss some things I've noticed and experienced having successfully used nod-don to get my 20.67 single UK NR last weekend. On my [YouTube channel](https://www.youtube.com/channel/UCX4LzNj0n6Pq1VfAjglln0A) I have two videos from the competition, one being my NR single and the other being a 24.65 from the previous round which gives a much better angle of the usage of nod-don than my NR does.
First, I spoke with Jules at the competition after he had seen the video of my record, where I had a decent discussion with him regarding my thoughts on this, which I'll now put on record.
TL;DR - Despite what appears to be popular belief, I think this should be disallowed in the future. I'm really on the fence but I think if it came down to it I'd have to err on the side of banning this technique. That being said, I definitely appreciate the reasons given by both sides on this argument which makes it really difficult to come to this conclusion, and I strongly believe that this is something that the community and the WCA should have a dialogue regarding over the coming months to try and determine the true place of nod-don in official speedsolving.
**The ambiguity that already technically exists between the end of the memorisation phase and the beginning of the solving phase is drastically increased using ND.**
The argument I see brought up a lot when somebody mentions the ambiguity that ND causes is that "regular 3BLD has this ambiguity already since somebody could make moves with one hand whilst donning the blindfold with the other". Whilst I accept this argument I think that comparing the two situations side by side is a little disingenuous. On the one hand, you are presented with the ability to start your execution with your hands already holding the cube exactly as you need to perform your first commutator at full speed as soon as the blindfold is donned. On the other, you get to perform maybe two or three moves maximum (and considering most top BLD solvers solve edges first using commutators that involve E/S/M moves this really isn't feasible at all as a time saving strategy) whilst in the process of returning your other hand to the puzzle. I feel like if there was a way to leverage this to your advantage then people would have done so by now (although you never really know considering this is the first time ND has been properly practiced by World Class BLD solvers such as Jack and Graham).
What are the consequences of this elevated ambiguity? I think they are far too concerning to ignore.
Some preface to this: The people who are the top of their game are implementing ND because saving even 0.3 seconds at sub-25 level is an insane amount of time, especially for something so minor. We're talking about people who learn entire new buffers to save fractions of seconds of time here. People look to these solvers as role models and will begin to implement ND themselves even if they average upwards of 2 minutes. In the same way that people who average above 5 minutes on 7x7 go and buy the Spark when they're currently using the Hays7, the optimisations made by the elite solvers trickle down into the whole community.
We are going to see an influx in competitors doing this all across the ability spectrum. This raises different issues at different levels:
•	For Elite solvers, the only way that a judge could be absolutely certain that the blindfold was fully donned before the competitor makes a move is if the competitor makes a noticeable pause between donning and executing – sacrificing a pretty large portion of the time that even doing ND in the first place saves. For ND to be optimised, it will have to require a precise hovering over the barrier of acceptable/unacceptable, something that a judge in the heat of the moment is almost certain to miss, especially where the don is subtle and not a vigorous shake of the head. I’m least concerned about this group as top solvers tend to record their solves, so video playback would immediately answer our questions, but this isn’t compulsory. Even the most experienced judges with the most trained eyes will have difficulty properly seeing whether ND was performed legally, and being the one to make the decision to deny a top solver their result because of what you may/may not have seen is a hell of a lot of pressure to put on somebody (especially if the solver were to leverage their status against the judge to tell them that “the don was fine, you must be seeing things”).
•	For intermediate and slower solvers, this is going to depend far more on competitor’s ability to adequately perform a don, and the judge’s concentration/understanding of the regulations. One of the numerous problems I had in practice was getting the blindfold to the perfect looseness that it would stay up whilst on my head but could be moved with the slightest of nods. If some competitor were to accidentally have their blindfold fall during memo, an inexperienced judge may place the card under them expecting them to start solving, or perhaps even DNF them if the competitor decided to lift the blindfold to continue memorisation. In the case where a judge isn’t paying much attention, they may not even register that the don has occurred and will not only be late to placing the blocker, but be incapable of determining whether the blindfold was fully donned before the solver began turning. I think this could even be an issue in top level bigBLD – judges are effectively volunteers who are taking their own time to watch other people solve, it’s very easy to get distracted and should someone do so at the worst time, they could very easily miss the ND and have the same issues as above. I haven’t heard about this technique being used in bigBLD as of yet but I would be surprised if Graham wasn’t doing so already.
In order to enforce the regulations in their current state in an environment where most solvers are using ND, it will require a massive overhead on the staff/delegates at a competition. Not only will judges have to watch solvers like hawks, but delegates will have to make calls on he-said-she-said scenarios. We already have so many issues with 3BLD in competitions with regards to judges not placing the blockers correctly and misscrambles being insanely more damaging than normal. It feels like something like ND will simply add fuel to the clusterfuckfire that is BLD events from a regulatory standpoint. I worry that this will discourage people from holding the event as it becomes a bigger and bigger ball-ache to deal with. 
This is something, however, I think that the WCA needs to discuss with the community either way. This technique has such wide-reaching implications that affect many different people for a ton of different reasons. I think this would be a great opportunity for WAC to get started and give the community a voice.

Thanks for reading

~Chris

**Comment by mark49152** [comment]
A couple of posts back I said it doesn't matter how the blindfold is donned as long as it's down before the first move, and Lucas pointed out that the issue is whether it's properly donned or not.  Having watched a few solves and talked to others about it, I agree with Lucas and Chris.  Nod don puts too much responsibility on the judge to pay close attention and make a potentially difficult split-second call on whether the blindfold is fully donned before the first move.  There will inevitably be lots of incidents, as well as undetected violations.  If anything, the regs should require a hand to be used to don, because that at least prevents the cube from being turned until they've finished moving the blindfold.

**Comment by Sixstringcal** [comment]
If the hands are required to be used to don, couldn't someone don it with their hands with the cube still in both hands making it EVEN MORE difficult for a judge to tell since the cube would be moving?  I feel like that would be more advantageous than not doing it, but less advantageous than the nod don.  By disallowing the nod don, I feel that it would make room for other optimizations that are more janky and more difficult to judge as the optimizations will be made regardless.

**Comment by xsrvmy** [comment]
I fail to see how requiring hand to don fixes anything. If anything the competitor can still perform an oh turn while doning

**Comment by Epride** [comment]
I think the issue with the "Nod-Don" is that it blurs the lines way too much between the memorisation and execution phase. While yes a competitor could perform an one handed turn while donning the blindfold I think the line is much less blurred here between the two phases. While some competitors who use the "Nod-Don" have a clear pause between nodding ans then turning what happens when those two become inseparable to the human eye. I am in favour of requiring a hand to don the blindfold as when competitors do this I have not experienced a case in which it is borderline if the blindfold was donned before the execution phase has started

**Comment by lgarron** [comment]
Okay, I'm seeing two problems:

- The nod don *may not consistently cover the competitor's eyes*.
- The nod don *does not clearly signal the beginning of the solve*.

I think the former could be solved by better blindfolds, and the latter could be solved by other signals such as "I'm starting the blindfolded phase!". However, we will inevitably have to deal with situations where neither is clearly done.

The best I can think of right now is to allow a nod don "at the discretion of the judge", but that's not really an improvement on the current situation.

The second-best would be to require touching the blindfold, but I can see issues with that – what's to stop a dangling string to pull it down?

I'd love any concrete suggestions on changes that are practical to make.

**Comment by lgarron** [comment]
By the way, I don't think that a nod don is inherently unacceptable – cubers do lots of things in pursuit of speed, and the only *real* problem here is ensuring that the competitor's vision is clearly obstructed during the blindfolded phase.

**Comment by mark49152** [comment]
@Lucas:  I think the start of the solving phase should be defined as the first move, and not anything to do with the blindfold.  That first move is the point of no return.  The competitor must ensure their blindfold is on before the first move.  If we require a hand to be taken off the cube and used to don the blindfold, even if via a string, that makes it easier for the judge to satisfy themselves that the blindfold was fully on before the first move.  If both hands are allowed to remain on the cube we will always have the problem that competitors will push the limits and start turning as soon as they can, even if the blindfold is still falling, leading to difficult situations for judges, inconsistent decisions, and unfairness.

**Comment by Samuel-Baird** [comment]
Another instance of an edge case. When looking at it frame by frame, one frame you can see the cube starting to be turned (not over 45 degrees) and the blindfold isn’t fully donned and the next frame the blindfold is donned and the cube is turned.  

https://youtu.be/HR3PuRttVEs

Nod-don is fine in many cases but it can definitely be hard for the judge to tell if the competitor has or hasn’t finished donning the blindfold before applying moves. The fact that it may not even be possible to tell when looking at video footage frame by frame for 30fps to me demonstrates that we are asking judges too much to fairly judge bld. I’m definitely in favor of making it illegal. 

Edit: here is a case where it should be a dnf but it’s hard to tell unless using video https://m.youtube.com/watch?v=BjK4jXM_bJE&feature=youtu.be
